README.md

```diff
@@ -62,52 +61,17 @@ import fs from "node:fs";
   const ephemeralReaderKey = Buffer.from(ephemeralReaderKeyHex, "hex");
 
   const trustedCerts = [fs.readFileSync("./caCert1.pem") /*, ... */];
-  const verifier = new Verifier(trustedCerts);
-  const mdoc = await verifier.verify(encodedDeviceResponse, {
+  const mdoc = await Verifier.verify(encodedDeviceResponse, {
     ephemeralReaderKey,
     encodedSessionTranscript,
   });
-
-  //at this point the issuer and device signature are valids.
-  inspect(mdoc);
-})();
-```
-
-### Getting diagnostic information
-
-```javascript
-import { Verifier } from "@animo-id/mdoc";
-import { inspect } from "node:util";
-import fs from "node:fs";
-
-(async () => {
-  const encodedDeviceResponse = Buffer.from(encodedDeviceResponseHex, "hex");
-  const encodedSessionTranscript = Buffer.from(
-    encodedSessionTranscriptHex,
-    "hex"
-  );
-  const ephemeralReaderKey = Buffer.from(ephemeralReaderKeyHex, "hex");
-
-  const trustedCerts = [fs.readFileSync("./caCert1.pem") /*, ... */];
-  const verifier = new Verifier(trustedCerts);
-
-  const diagnosticInfo = await verifier.getDiagnosticInformation(
```

Comment:
is this not supported anymore? It can be quite helpful

Reply:
I removed it, because I had to change a lot of the code again to make it work. If we use it, I can add it back in a follow up PR, but I didn't know we used it.